src/app/services/mail/mail.service.ts

```diff
@@ -156,19 +156,35 @@ export class MailService {
       ...loggerMeta,
     }
 
+    const sendMailPromise = new Promise<true>((resolve, reject) => {
+      this.#transporter.sendMail(mail, (error, info) => {
+        // Log SES message ID
+        logger.info({
+          message: 'this.#transporter.sendMail',
+          meta: {
+            ...logMeta,
+            action: 'this.#transporter.sendMail',
+            info,
+          },
+        })
+        if (error) reject(error)
+        resolve(true)
+      })
+    })
+
     return promiseRetry<true>(async (retry, attemptNum) => {
       logger.info({
         message: `Attempt ${attemptNum} to send mail`,
         meta: logMeta,
       })
 
       try {
-        await this.#transporter.sendMail(mail)
+        const result = await sendMailPromise
```

Comment:
Might be easier to chain the promise instead:

```ts
try {
  this.#transporter.sendMail(mail).then(info => {
    // Log SES message ID
    logger.info({
      message: 'this.#transporter.sendMail',
      meta: {
        ...logMeta,
        action: 'this.#transporter.sendMail',
        info,
      },
    })
    return true
  })
} catch (error) { ... }
```

Reply:
thanks; simplified further, for review